### ISSUE TITLE
Outline on < > buttons 

### ISSUE THREAD

**Issue by crianonim** [fac-17/Plantbase-Me#227]
Not sure if it actually should be removed (accessibility) but if you do want to remove it's as simple as:
outline:none
in css
